src/ResourceManager/AzureBatch/Commands.Batch/AzureRM.Batch.psd1

```diff
@@ -147,11 +147,12 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Fixed issue with default resource groups not being set.
-* Updated common runtime assemblies'
+        ReleaseNotes = '* Added the ability to see what version of the Azure Batch Node Agent is running on each of the VMs in a pool, via the new `NodeAgentInformation` property on `PSComputeNode`.
+* Removed the `ValidationStatus` property from `PSTaskCounts`.
+* The `Caching` default for `PSDataDisk` is now `ReadWrite` instead of `None`.'
 
         # Prerelease string of this module
-        # Prerelease = ''
+        Prerelease = 'preview'
```

Comment:
Ah, need to fix this

Reply:
Oh, is this the right file? Or should I delete this file and just update the Az.Batch.psd1?

Reply:
I fixed this but not sure if this file should just be deleted

Reply:
Brought this back and now the tests are passing